packages/kit/src/components.ts

```diff
@@ -9,12 +9,12 @@ import { logger } from './logger'
  *
  * Requires Nuxt 2.13+
  */
-export async function addComponentsDir (dir: ComponentsDir) {
+export async function addComponentsDir (dir: ComponentsDir, insertMethod: 'push' | 'unshift' = 'push') {
```

Comment:
Instead, can we support an options object with `prepend` key, in line with other kit utilities?

https://github.com/nuxt/nuxt/blob/1444d89e3b6170eaed56c16494b1ff6b9b68ef9e/packages/kit/src/imports.ts#L14